### DISCUSSION TITLE
how to generate other language test report(DevTool)

### DISCUSSION THREAD

**Discussion by pyg955** [GoogleChrome/lighthouse#14734] [Q&A]
The developer tool has set the language to Chinese, but the test reports generated using the lighthouse are still in English (however, another friend of mine can generate Chinese test reports using the lighthouse in the developer tool).

**Comment by adamraine** [comment]
Setting the language in DevTools should change the language of the Lighthouse report

![Screen Shot 2023-01-30 at 11 03 15 AM](https://user-images.githubusercontent.com/6752989/215571668-7e0ceae2-476a-40a0-a28e-1e8a394d50e3.png)
